tests/fft_test.py

```diff
@@ -353,5 +353,33 @@ def testRfftfreqErrors(self, n):
       lambda: func(n=10, d=n)
     )
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+    {"testcase_name": "_shape={}".format(
+      jtu.format_shape_dtype_string(shape, dtype)),
+      "dtype": dtype, "shape": shape, "rng_factory": rng_factory}
+    for rng_factory in [jtu.rand_default]
+    for dtype in all_dtypes
+    for shape in [[9], [10], [101], [102], [3, 5], [3, 17], [5, 7, 11]]))
```

Comment:
Looking over these tests, I noticed that there does not seems to be any test coverage for providing the `axes` argument. Could you please add that?